# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/LocalAccessible.h
# Commit: 5d76535b47fc
# Full Hash: 5d76535b47fced08e46753a0262211732fb26417
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P7: Move and rename GetPositionAndSizeInternal to Accessible. r=Jamie
#   
#   Now that the AccGroupInfo getter is in Accessible, the position and size
#   getter could go there too.
#   
# ==============================================================================

diff -r fca1ff7eabf4 -r 5d76535b47fc accessible/generic/LocalAccessible.h
--- a/accessible/generic/LocalAccessible.h	Tue Jan 04 21:01:36 2022 +0000
+++ b/accessible/generic/LocalAccessible.h	Tue Jan 04 21:01:37 2022 +0000
@@ -272,16 +272,6 @@
   virtual LocalAccessible* FocusedChild();
 
   /**
-   * Calculate position in group and group size ('posinset' and 'setsize') based
-   * on accessible hierarchy.
-   *
-   * @param  aPosInSet  [out] accessible position in the group
-   * @param  aSetSize   [out] the group size
-   */
-  virtual void GetPositionAndSizeInternal(int32_t* aPosInSet,
-                                          int32_t* aSetSize);
-
-  /**
    * Get the relation of the given type.
    */
   virtual Relation RelationByType(RelationType aType) const;